apps/prairielearn/src/pages/instructorFileBrowser/instructorFileBrowser.html.ts

```diff
@@ -1,39 +1,146 @@
 import { filesize } from 'filesize';
 
-import { escapeHtml, html, joinHtml, unsafeHtml } from '@prairielearn/html';
+import { escapeHtml, html, type HtmlValue, joinHtml, unsafeHtml } from '@prairielearn/html';
 import { renderEjs } from '@prairielearn/html-ejs';
 
 import { compiledScriptTag, nodeModulesAssetPath } from '../../lib/assets.js';
 import { config } from '../../lib/config.js';
+import { User } from '../../lib/db-types.js';
+import type { Paths } from '../../lib/instructorFiles.js';
```

Comment:
Can we call this type something like `InstructorFilePaths` or just generally something less generic than `Paths`?

Reply:
Done in 87335d6d7.